src/cljs/main/broadfcui/page/workspace/summary/tab.cljs

```diff
@@ -315,6 +323,11 @@
           (swap! state update :server-response assoc :server-error err-text)
           (swap! state update :server-response
                  assoc :billing-projects (map #(% "projectName") projects)))))
+     (endpoints/get-library-groups
+      (fn [{:keys [success? get-parsed-response]}]
+        (if success?
+          (swap! state update :server-response assoc :library-groups (get-parsed-response))
+          (swap! state update :server-response assoc :server-error "Unable to load library groups"))))
```

Comment:
yeah, i was thinking since it really doesn't change much we should do it a few times as possible (i.e. not every time they open the catalog wizard). but i see that there are many times a user might go to this page and never open the wizard - and even if they do, they probably won't open it multiple times. will move.